.env.example

```
@@ -1,5 +1,11 @@
 # Crawlerr FastMCP Server Configuration
 
+# Docker Service Ports
+QDRANT_HTTP_PORT=7000
+QDRANT_GRPC_PORT=7001
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The QDRANT_GRPC_PORT key should go before the QDRANT_HTTP_PORT key

(UnorderedKey)